app/models/spree/authentication_method.rb

```diff
@@ -1,19 +1,27 @@
 # frozen_string_literal: true
 
-class Spree::AuthenticationMethod < ApplicationRecord
-  def self.provider_options
-    SolidusSocial.configured_providers.map { |provider_name| [provider_name.split("_").first.camelize, provider_name] }
-  end
-
-  validates :provider, presence: true
+class Spree::AuthenticationMethod < ActiveRecord::Base
+  validates :provider, :api_key, :api_secret, presence: true
```

Comment:
Is this storing the credentials unencrypted in the database or do we have some mechanism to store them securely? If so, is this documented somewhere?

This should be addressed before we publish a new version to Rubygems. 

/cc @fthobe